packages/pps/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,94 @@
+---
+description: Pipeline for processing Pleasant Password Server logs.
+processors:
+        - rename:
+                  field: message
+                  target_field: event.original
+                  ignore_missing: true
+                  if: ctx.event?.original == null
+        - set:
+                  field: ecs.version
+                  value: "8.11.0"
+        - grok:
+                  field: event.original
+                  patterns:
+                          - "^<%{NUMBER:log.syslog.priority:long}>%{SYSLOGTIMESTAMP:event.created}\\s+%{NOTSPACE:host.hostname}\\s%{DATA}:%{IP:client.ip}\\s-\\s%{USERNAME:user.name}@%{DATA:user.domain}\\s%{DATA}(?<event.outcome>(Success)|(Error))\\s-\\s%{GREEDYDATA:message}"
+                          - "^<%{NUMBER:log.syslog.priority:long}>%{SYSLOGTIMESTAMP:event.created}\\s+%{NOTSPACE:host.domain}\\s%{DATA}:%{IP:client.ip}%{DATA}(?<event.outcome>(Success)|(Error))\\s-\\s%{GREEDYDATA:message}"
```

```suggestion
                          - '^<%{NUMBER:log.syslog.priority:long}>%{SYSLOGTIMESTAMP:event.created}\s+%{NOTSPACE:host.hostname}\s%{DATA}:%{IP:client.ip}\s-\s%{USERNAME:user.name}@%{DATA:user.domain}\s%{DATA}(?<event.outcome>(Success)|(Error))\s-\s%{GREEDYDATA:message}'
                          - '^<%{NUMBER:log.syslog.priority:long}>%{SYSLOGTIMESTAMP:event.created}\s+%{NOTSPACE:host.domain}\s%{DATA}:%{IP:client.ip}%{DATA}(?<event.outcome>(Success)|(Error))\s-\s%{GREEDYDATA:message}'
```
to reduce escape spam.